pkgs/development/python-modules/seafobj/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, fetchpatch, fetchFromGitHub, python3Packages, makeWrapper }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "seafobj";
+  version = "unstable-2021-07-08";
+
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = "seafobj";
+    rev = "df13a98e7e32c926083ca60bb7fb1bbc4dfcdbd0"; # using a fixed revision because upstream may re-tag releases :/
+    sha256 = "0jwgbx083mwwh18x7450igq748bbld9s7zgibwv1cxwrrny1aari";
+  };
+
+  patches = [
+    ./setuptools.patch
+  ];
+  doCheck = false; # disabled because it requires a ccnet environment
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  meta = with lib; {
+    homepage = "https://github.com/haiwen/seafobj";
+    description = "python library for accessing seafile data model";
```

```suggestion
    description = "Python library for accessing seafile data model";
```
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes